internal/sdks/sdks.go

```diff
@@ -23,21 +27,33 @@ func ReplaceFlagKey(instructions string, key string) string {
 	return r.Replace(instructions)
 }
 
-// ReplaceSDKKey changes the placeholder flag key in the SDK instructions to the flag key from
-// the user.
-func ReplaceSDKKey(instructions string, key string) string {
+// ReplaceSDKKeys changes the placeholder SDK key/client side ID in the SDK instructions to the key from
+// the default test environment for the user's account.
+func ReplaceSDKKeys(instructions string, sdkKey, clientSideId string) string {
```

Comment:
updated this function name and signature to update both the SDK key and client side ID